projects/kit/components/input-phone-international/index.ts

```diff
@@ -1,7 +1,10 @@
-export * from './const/mask-after-code-regexp';
 export * from './input-phone-international.component';
 export * from './input-phone-international.module';
 export * from './input-phone-international.options';
-export * from './tokens/countries-masks';
 export * from './utils/extract-value-from-event';
-export * from './utils/not-kz-region';
+/* TODO drop in v4.0 */
+export {tuiNotKzRegion} from '@taiga-ui/kit/utils/miscellaneous';
```

Comment:
I'm not sure is't good

<img width="844" alt="image" src="https://user-images.githubusercontent.com/12021443/234503041-ab0d2d80-19bc-4cbc-801f-7043ad2e983a.png">

Reply:
How to solve it? because someone could import these constants in the project from `@taiga-ui/kit/components/input-phone-international`, and if you just move them to another place it will be breaking changes.

Reply:
Deep imports are considered wrong with Taiga UI, it is equivalent to importing from `@taiga-ui/kit` in terms of tree shaking. So I would say it is not breaking change.